playbooks/roles/nodes-config/tasks/main.yaml

```diff
@@ -6,7 +6,7 @@
   wait_for_connection:
     delay: 15
     connect_timeout: 15
-    timeout: 1800
+    timeout: 3600
```

Comment:
This is made configurable and will be implemented in the powervm repo. The default in master is set to 2700 (45m).

Reply:
+1